.pre-commit-config.yaml

```diff
@@ -10,14 +10,23 @@ repos:
       - id: flake8
         language_version: python3
   - repo: https://github.com/pycqa/isort
-    rev: 5.7.0
+    rev: 5.10.1
```

Comment:
Any reason why we're bumping isort's pre-commit hook?

If we decide to keep this, we should also bump the version used in the conda environments and setup.py